nmdc_automation/config/workflows/workflows-mt.yaml

```diff
@@ -57,30 +57,30 @@ Workflows:
         description: "rRNA fastq for {id}"
 
   - Name: Metatranscriptome Reads QC Interleave
-    Type: nmdc:ReadQcAnalysisActivity
+    Type: nmdc:ReadQcAnalysis
     Enabled: True
     Analyte Category: Metatranscriptome
     Git_repo: https://github.com/microbiomedata/metaT_ReadsQC
-    Version: v0.0.7
-    Collection: read_qc_analysis_activity_set
+    Version: v0.0.3
```

Comment:
merge conflict kept the wrong version, this should be v0.0.7

Reply:
https://github.com/microbiomedata/nmdc_automation/pull/253/commits/01235e56038232ac22c09a20944a3b449210a258